src/plot_api/plot_config.js

```diff
@@ -83,8 +84,10 @@ module.exports = {
     setBackground: defaultSetBackground,
 
     // URL to topojson files used in geo charts
-    topojsonURL: 'https://cdn.plot.ly/'
+    topojsonURL: 'https://cdn.plot.ly/',
 
+    // Turn all console logging on or off (errors will be thrown)
+    logging: false
```

Comment:
So `logging` is a little different than the other existing plot config options.

Plot config options are meant to set global `Plotly.plot` options that aren't affected by `restyle` or `relayout`.

Instead `logging` is really a `Plotly` option.

We don't need to differentiate between the two right now, but @mdtusz could add a comment above `logging` mentioning that it can **only** be set using `Plotly.setPlotConfig` /